types/apollo-upload-client/index.d.ts

```diff
@@ -1,22 +1,52 @@
-// Type definitions for apollo-upload-client 8.1
+// Type definitions for apollo-upload-client 14.1
 // Project: https://github.com/jaydenseric/apollo-upload-client#readme
-// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>, tyankatsu <https://github.com/tyankatsu0105>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.6
 
-import { ApolloLink } from "apollo-link";
-import { HttpOptions } from "apollo-link-http-common";
+// ==============================================================================
+// imports
+// ==============================================================================
 
-export { ReactNativeFile } from "extract-files";
+import { ApolloLink } from '@apollo/client/core';
```

Comment:
Any reason why this is not imported from `apollo-link`?
Looks like both are not exactly the same because it seems to create an issue in my project.

```
Type 'import("xxxx/node_modules/@apollo/client/link/core/ApolloLink").ApolloLink' is not assignable to type 'import("xxxx/node_modules/apollo-link/lib/link").ApolloLink'.
  Types of property 'split' are incompatible.
    Type '(test: (op: import("xxxx/node_modules/@apollo/client/link/core/types").Operation) => boolean, left: import("xxxx/node_modules/@apollo/client/link/core/ApolloLink").ApolloLink | import("xxxx...' is not assignable to type '(test: (op: import("xxxx/node_modules/apollo-link/lib/types").Operation) => boolean, left: import("xxxx/node_modules/apollo-link/lib/link").ApolloLink | import("xxxx/node_modules/apollo-li...'.
      Types of parameters 'test' and 'test' are incompatible.
        Types of parameters 'op' and 'op' are incompatible.
          Property 'toKey' is missing in type 'import("xxxx/node_modules/@apollo/client/link/core/types").Operation' but required in type 'import("xxxx/node_modules/apollo-link/lib/types").Operation'.ts(2322)
```